examples/demo/src/index.ts

```diff
@@ -1,14 +1,41 @@
 import './index.css';
-import { editor, Environment, languages } from 'monaco-editor';
+import { parse } from 'jsonc-parser';
+import { editor, Environment, languages, Uri } from 'monaco-editor';
 import { configureMonacoTailwindcss, tailwindcssData } from 'monaco-tailwindcss';
+import { TailwindConfig } from 'tailwindcss/tailwind-config';
 
 declare global {
   interface Window {
     MonacoEnvironment: Environment;
   }
 }
 
-configureMonacoTailwindcss();
+const initialConfig: TailwindConfig = {
+  theme: {
+    extend: {
+      colors: {
+        // https://icolorpalette.com/color/molten-lava
+        lava: '#b5332e',
+        // Taken from https://icolorpalette.com/color/ocean-blue
+        ocean: {
+          50: '#f2fcff',
+          100: '#c1f2fe',
+          200: '#90e9ff',
+          300: '#5fdfff',
+          400: '#2ed5ff',
+          500: '#00cafc',
+          600: '#00a3cc',
+          700: '#007c9b',
+          800: '#00546a',
+          900: '#002d39',
+        },
+      },
+    },
+  },
+  plugins: [],
```

Comment:
we dont support this yet (and will likely not support plugins right?)

Reply:
I just copied this from the tailwind installation guide and extended it, but I think you’re right. I’ll remove it.